src/components/Modal/UpdateProposalStatusModal/UpdateProposalStatusModal.tsx

```diff
@@ -60,30 +66,69 @@ const getPrimaryButtonTextKey = (status?: ProposalStatus | null) => {
 }
 
 export function UpdateProposalStatusModal({
-  onClickAccept,
   proposal,
+  isDAOCommittee,
   status,
   loading,
   open,
+  proposalKey,
   className,
   ...props
 }: Props) {
   const t = useFormatMessage()
+  const isOneTimePaymentProposalTier = GrantTier.isOneTimePaymentTier(proposal?.configuration.tier)
+  const queryClient = useQueryClient()
+
+  const onClose = () => {
+    setError('')
```

Comment:
Should we also reset the form field values too?

Reply:
I don't think it's necessary